pkg/providers/imagefamily/ubuntu_2204.go

```diff
@@ -94,8 +94,8 @@ func (u Ubuntu2204) ScriptlessCustomData(kubeletConfig *bootstrap.KubeletConfigu
 			CABundle:         caBundle,
 			GPUNode:          u.Options.GPUNode,
 			GPUDriverVersion: u.Options.GPUDriverVersion,
-			GPUImageSHA:      u.Options.GPUImageSHA,
 			GPUDriverType:    u.Options.GPUDriverType,
+			GPUImageSHA:      u.Options.GPUImageSHA,
```

Comment:
why

Reply:
Aligning with azure linux image family. Found it a little annoying when doing the diff, so felt worth fixing, since less than 5 min change.